python/cmd/data.py

```diff
@@ -42,9 +42,12 @@
 from argparse import ArgumentParser
 import importlib
 
-from housinginsights.config.base import HousingInsightsConfig
+import sys, os
+sys.path.append(os.path.abspath('../'))
```

Comment:
If you change this to `sys.path.append(os.path.abspath('../../'))` you can omit the python in front of the import statements (e.g. `just from housinginsights.config.base import HousingInsightsConfig)(